src/library/crate/cratestorage.cpp

```diff
@@ -538,7 +538,7 @@ CrateTrackSelectResult CrateStorage::selectTracksSortedByCrateNameLike(const QSt
                     CRATETABLE_ID,
                     CRATETRACKSTABLE_CRATEID,
                     CRATETABLE_NAME));
-    query.bindValue(":crateNameLike", kSqlLikeMatchAll + crateNameLike + kSqlLikeMatchAll);
+    query.bindValue(":crateNameLike", QVariant(kSqlLikeMatchAll % crateNameLike % kSqlLikeMatchAll));
```

Comment:
kSqlLikeMatchAll can also be initialized with L'%‘

Does QStringBuilder support QChar or do we create a temporary QString here anyway? In the later case we my considder to define 
kSqlLikeMatchAll as QString

Reply:
There is a template specialization of QConcatenable for all kinds of chars.